portfolio/src/main/java/com/google/sps/servlets/DeleteCommentServlet.java

```diff
@@ -29,13 +32,32 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet responsible for deleting tasks. */
+/** 
+* Servlet that deletes comment entities
+*
+* @param request which contains data used to identify a comment in order to
+* delete it
+* @param response
+*/
 @WebServlet("/delete-comment")
 public class DeleteCommentServlet extends HttpServlet {
 
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    Query query = new Query("Comment");
+    // Receive input on which blog we are retrieving comments from
+    int maxNumBlogs = 5;
```

Comment:
This looks like a constant, you can extract it to the class level and mark it static final:

`static final int MAX_NUM_BLOGS = 5;`